framework/test/common/application.js

```diff
@@ -15,91 +15,100 @@
 'use strict';
 
 // Global imports
-const dns = require('dns');
-const net = require('net');
-const Promise = require('bluebird');
+const util = require('util');
 const rewire = require('rewire');
 const async = require('async');
-const httpApi = require('../../src/modules/chain/helpers/http_api');
+const express = require('express');
+const ed = require('../../src/modules/chain/helpers/ed.js');
 const jobsQueue = require('../../src/modules/chain/helpers/jobs_queue');
 const Sequence = require('../../src/modules/chain/helpers/sequence');
 const { createCacheComponent } = require('../../src/components/cache');
 const StorageSandbox = require('./storage_sandbox').StorageSandbox;
+const ZSchema = require('../../src/modules/chain/helpers/z_schema.js');
+const initSteps = require('../../src/modules/chain/init_steps');
 
+const promisifyParallel = util.promisify(async.parallel);
 let currentAppScope;
 
+const modulesInit = {
+	accounts: '../../src/modules/chain/modules/accounts.js',
+	blocks: '../../src/modules/chain/modules/blocks.js',
+	dapps: '../../src/modules/chain/modules/dapps.js',
+	delegates: '../../src/modules/chain/modules/delegates.js',
+	loader: '../../src/modules/chain/modules/loader.js',
+	multisignatures: '../../src/modules/chain/modules/multisignatures.js',
+	peers: '../../src/modules/chain/modules/peers.js',
+	rounds: '../../src/modules/chain/modules/rounds.js',
+	signatures: '../../src/modules/chain/modules/signatures.js',
+	system: '../../src/modules/chain/modules/system.js',
+	transactions: '../../src/modules/chain/modules/transactions.js',
+	transport: '../../src/modules/chain/modules/transport.js',
+};
+
 function init(options, cb) {
 	options = options || {};
 	options.scope = options.scope ? options.scope : {};
 	// Wait for genesisBlock only if false is provided
 	options.scope.waitForGenesisBlock = options.waitForGenesisBlock !== false;
 
-	if (options.sandbox) {
-		const storage = new StorageSandbox(
-			options.sandbox.config || __testContext.config.db,
-			options.sandbox.name
-		);
-		storage.bootstrap().then(() => {
-			options.scope.storage = storage;
-			__init(options.scope, cb);
-		});
-	} else {
-		__init(options.scope, cb);
-	}
+	__init(options.sandbox, options.scope)
+		.then(scope => cb(null, scope))
+		.catch(err => cb(err));
 }
 
 // Init whole application inside tests
-function __init(initScope, done) {
+async function __init(sandbox, initScope) {
```

Comment:
Are we using double `_` or just one? I've been seeing single `_` for private methods everywhere but `__` for private variables at the same time.

Reply:
As of now we are using double underscore notation in core. So we should stick wit it.